### PR TITLE
more useful example

### PR DESCRIPTION
**Pull request by kkraune**

### CODE REVIEW

File: documentation/document-api-guide.html [hunk 1]
```diff
@@ -50,14 +48,75 @@ <h2 id="documents">Documents</h2>
...
+
+        // Update document with a1,10 and a2,20
+        DocumentUpdate upd1 = new DocumentUpdate(type, id);
+        WeightedSet&lt;StringFieldValue&gt; wset1 = new WeightedSet&lt;&gt;(DataType.getWeightedSet(DataType.STRING));
+        wset1.put(new StringFieldValue("a1"), 10);
+        wset1.put(new StringFieldValue("a2"), 20);
+        upd1.addFieldUpdate(FieldUpdate.createAddAll(type.getField("aWeightedset"), wset1));
+        System.out.println(upd1.toString());
+        session.update(upd1);
+
```
Comment (vekterli) [inline documentation/document-api-guide.html]: Could be nice to have a final step here where we Get the document back out again and print it, showing that the operations have taken effect.

File: documentation/document-api-guide.html [hunk 2]
```diff
@@ -179,10 +238,10 @@ <h2 id="document-updates">Document updates</h2>
         new IntegerFieldValue(1), // array index
         new AssignValueUpdate(new IntegerFieldValue(2))));  // value at index
 
-// Composite - update weightedset element
-upd.addFieldUpdate(FieldUpdate.createMap(type.getField("aWeightedset"),
+// DO NOT USE - createMap can cause crash / missing updates TODO FIXME
```
Comment (vekterli) [inline documentation/document-api-guide.html]: Hopefully we'll remember to update this once the bug is fixed 😉

File: documentation/document-api-guide.html (hunk 1 above)
Comment (kkraune) [reply documentation/document-api-guide.html]: great idea, will add in next iteration !

File: documentation/document-api-guide.html (hunk 2 above)
Comment (kkraune) [reply documentation/document-api-guide.html]: I'll consider making it a publish issue for easier reference

**Pull request merged**